org-super-agenda.el

```diff
@@ -647,6 +647,30 @@ available."
           (_ (cl-loop for fn in args
                       thereis (funcall fn item)))))
 
+(org-super-agenda--defgroup property
+  "Group items that contain a property with a given value.
```

Comment:
I think the description can be more concise.  How about:
```el
  "Group items with a property, optionally matching a value.
Argument may be a property name string, or a list of property
name string and either value string or predicate with which to
test the value."
```

Reply:
Good idea - fixed!